app/models/scanned_map.rb

```diff
@@ -15,11 +15,16 @@ def self.can_have_manifests?
     true
   end
 
+  # How is this used?
```

Comment:
This is a good question - I'm not sure it is anymore. It used to be the label on display in a lot of places for Hyrax, but I think that's a different method in the decorator now.